Makefile

```diff
@@ -56,23 +55,6 @@ all: $(addsuffix /all,$(MAKEDIRS))
 
 init:
 
-$(CODE_REVIEWER_LOCATION):
```

Comment:
maybe add one sentence to the PR, e.g. "we do not need codereviewr anymore, because we now use dora metrics in revolution and this just requires a github webhook, which is already up and running"

Reply:
done